plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/UiSessionTimeout.js

```diff
@@ -55,30 +64,16 @@ Ext.define('NX.controller.UiSessionTimeout', {
           afterrender: me.startTicking
         },
         'nx-expire-session button[action=cancel]': {
-          click: me.stopTicking
+          click: me.setupTimeout
         }
       }
     });
   },
 
   onLaunch: function () {
-    var me = this,
-        user = NX.State.getUser(),
-        uiSettings = NX.State.getValue('uiSettings') || {};
-
-    me.setupTimeout(user ? uiSettings['sessionTimeout'] : undefined);
-  },
-
-  /**
-   * @private
-   */
-  onUserChanged: function (user) {
-    var me = this,
-        uiSettings = NX.State.getValue('uiSettings') || {};
-
-    me.setupTimeout(user ? uiSettings['sessionTimeout'] : undefined);
+    this.setupTimeout();
```

Comment:
I don't see the point of introducing the temp variable me for the single call required, but totally agree that it makes js scoping+readability better when we need to access it more than once or in nested code blocks.